Makefile

```diff
@@ -17,7 +17,7 @@
 #    make install prefix=$HOME
 
 # -s WASM=0 -s SINGLE_FILE=1: allow loading zee.js synchronously
-EMCC=emcc -O2 -s WASM=0 -s SINGLE_FILE=1 -s EXPORTED_FUNCTIONS="['_gzopen', '_gzread', '_gzwrite', '_gzclose', '_gzerror', '_gzeof']" --pre-js pre.js --post-js post.js
+EMCC=emcc -O2 -s WASM=0 -s WASM_ASYNC_COMPILATION=0 -s SINGLE_FILE=1 -s EXPORTED_FUNCTIONS="['_gzopen', '_gzread', '_gzwrite', '_gzclose', '_gzerror', '_gzeof']" --pre-js pre.js --post-js post.js
```

Comment:
is the output file small enough to work in chrome?

Reply:
Isn't WASM=0 meant to deal with this?

Reply:
Oh, I missed that. Yeah, without wasm there isn't an issue with code size. However, then we can remove `WASM_ASYNC_COMPILATION` as it has no effect.

Reply:
`nodejs test.js` strongly disagrees :)
`Assertion failed: you need to wait for the runtime to be ready (e.g. wait for main() to be called)
`

Reply:
Ah, yes, I forgot - in upstream, wasm=0 tracks the wasm=1 path very closely, to make it easier to swap between the two. In particular, it looks like wasm all through the instantiation, all using a fake WebAssembly object. And that includes async startup. So yes, it is necessary there. Sorry for the mixup!

Reply:
Thanks -- it's somewhat reassuring to see I'm not the only one struggling to keep all this in mind ;)